src/boost/python/server.py

```diff
@@ -1088,11 +1126,12 @@ def Pressurize(self, pressure):
             green_mode=green_mode)
     name = f.__name__
 
-    dtype_in, dformat_in = _get_tango_type_format(dtype_in, dformat_in)
-    dtype_out, dformat_out = _get_tango_type_format(dtype_out, dformat_out)
 
-    din = [from_typeformat_to_type(dtype_in, dformat_in), doc_in]
-    dout = [from_typeformat_to_type(dtype_out, dformat_out), doc_out]
+    t_dtype_in, t_dformat_in = _get_tango_type_format(dtype_in, dformat_in)
+    t_dtype_out, t_dformat_out = _get_tango_type_format(dtype_out, dformat_out)
```

Comment:
Can we have more meaningful names for those variables? Or keep them packed as `typeformat_in` and `typeformat_out`?